image/db/rhel/scripts/custom-entrypoint.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+# The postgres server has been started once during the build process in the Dockerfile.
+# Now we need to start it again, but this time with the correct password.
+# So we need to issue a command to change the password.
+
+set -e
+
+if [ ! -d "/var/lib/postgresql/data/pgdata" ]; then
+
+  echo "Creating /var/lib/postgresql/data/pgdata..."
+  mkdir -p /var/lib/postgresql/data/pgdata
+
+  echo "Moving archive to target directory..."
+  mv /tmp/data.tar.gz /var/lib/postgresql/data/pgdata/data.tar.gz
+
+  echo "Uncompressing into /var/lib/pgsql/data/pgdata..."
+  tar -xzf /var/lib/postgresql/data/pgdata/data.tar.gz -C /var/lib/postgresql/data/pgdata
+
+  echo "Removing archive..."
+  rm /var/lib/postgresql/data/pgdata/data.tar.gz
```

Comment:
@dcaravel @RTann Could we move these steps to build time instead of start-up time?
@ludydoo proposes this to save IO in the cloud service which prevents us from cluster upgrades and increases ACS upgrade complexity.
Every restart of ACS instances causes to write ~1.8GB per instance.
Additionally the manifest must be updated that the scanner-db pod can read the Postgres Password.

What do you think about this approach? It seems simple and scaleable to us.
cc @ludydoo